scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -262,7 +262,7 @@
         (reduce (fn [task-ents-by-user task-ent]
                   (let [user (util/task-ent->user task-ent)
                         f (if (= new-running-task-ent task-ent)
-                            (fnil conj (sorted-set-by (util/same-user-task-comparator-penalize-backfill)))
+                            (fnil conj (sorted-set-by (util/same-user-task-comparator)))
```

Comment:
I don't think so. The `user->sorted-running-task-ents` will only have a user if it has at least one running job. In the rebalancer, we are considering "what would happen" if we let job X run. Job X may be from a user who has nothing currently running and therefore wouldn't exist in `user->sorted-running-task-ents`. 

Let me know if you had something else in mind.